python/dgl/mock_sparse/sp_matrix.py

```diff
@@ -171,7 +171,7 @@ def __call__(self, x):
         assert len(x) == self.nnz
         return SparseMatrix(self.row, self.col, x, shape=self.shape)
 
-    def indices(self, fmt, return_shuffle=False) -> Tuple[torch.tensor, ...]:
```

Comment:
`Torch.Tensor` instead of `torch.tensor`.

Reply:
Done.